sky/serve/replica_managers.py

```diff
@@ -182,6 +184,9 @@ class ProcessStatus(enum.Enum):
     # The process is finished and succeeded
     SUCCEEDED = 'SUCCEEDED'
 
+    # The process is interrupted
+    INTERRUPTED = 'INTERRUPTED'
```

Comment:
Could we add this in a separate PR?

Reply:
This is needed as we will interrupt a replica when it is initializing.